sci-geosciences/josm-bin/josm-bin-19207-r1.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop java-utils-2 xdg
+
+DESCRIPTION="Java-based editor for the OpenStreetMap project"
+HOMEPAGE="https://josm.openstreetmap.de/"
+# PV should be stable here https://josm.openstreetmap.de/wiki/StartupPage
+SRC_URI="
+	https://josm.openstreetmap.de/download/josm-snapshot-${PV}.jar
+	https://josm.openstreetmap.de/export/${PV}/josm/trunk/native/linux/tested/usr/share/applications/org.openstreetmap.josm.desktop -> ${P}.desktop
+	https://josm.openstreetmap.de/export/${PV}/josm/trunk/native/linux/tested/usr/share/mime/packages/josm.xml \
+		-> ${P}.mime.xml
```

Comment:
We might run into a verification problem if these files are changed at some point.

Can we download them from some place containing the PV? Maybe a versioned source tarball?

Reply:
both URLs point to versioned files and contain the PV and should be fine

There are two more possible sources, github releases (they seem to migrate over from trac) and the debian packages they build.

but i find that just downloading the two files from the currently official vcs seems to be the best option

Reply:
Resolving this thread since the src URI does contain PV and the dest also does with P, feel free to follow up in case i misunderstood the request.

Reply:
I agree that this is fine and well-versioned.

Reply:
Yeah sorry, my bad, I was clearly too tired when looking at this.